bin/rails

```diff
@@ -6,3 +6,9 @@ end
 APP_PATH = File.expand_path('../../config/application', __FILE__)
 require_relative '../config/boot'
 require 'rails/commands'
+
+if ENV['RAILS_ENV'] == 'test'
+  require 'simplecov'
+  SimpleCov.start 'rails'
+  puts "required simplecov"
```

Comment:
Don't use `puts` within your Rails application.  If you want to get info from your app, use `Rails.logger` (see comment above.)